src/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+
+#include "Server.h"
+
+constexpr int EXPECTED_ARGUMENTS = 2;
+
+int main(const int argc, char **argv)
+{
+    if (EXPECTED_ARGUMENTS != argc)
+    {
+        std::cout << "Usage: " << argv[0] << " [port]" << std::endl;
+        return EXIT_FAILURE;
+    }
+    const int server_port = std::stoi(argv[1]);
+    const size_t max_connections = 10;
```

Comment:
didnt u define max connections in the Def.h?

Reply:
No, I left it to be configurable from main